src/browser/modules/Editor/Editor.jsx

```diff
@@ -321,22 +354,22 @@ export class Editor extends Component {
             onClick={() => this.props.onFavortieClick(this.getEditorValue())}
             disabled={this.getEditorValue().length < 1}
             title='Favorite'
-            hoverIcon='"\58"'
-            icon='"\73"'
+            hoverIcon='&quot;\58&quot;'
+            icon='&quot;\73&quot;'
           />
           <EditorButton
             onClick={() => this.clearEditor()}
             disabled={this.getEditorValue().length < 1}
             title='Clear'
-            hoverIcon='"\e005"'
-            icon='"\5e"'
+            hoverIcon='&quot;\e005&quot;'
```

Comment:
That's certainly more uglier than before. It does however feel more explicit about how the string is going to be encoded